src/css/profile/mobile/common/popup.less

```diff
@@ -143,6 +156,49 @@
 		border-bottom-width: 1 * @px_base;
 		border-bottom-color: #e6e6e6;
 		border-bottom-style: solid;
+		margin-left: -16 * @px_base;
+	}
+
+	.ui-popup-notitle {
+		@media (orientation: portrait) and (max-width: 479px) {
+			margin-top: 24 * @px_base;
+		}
+		@media (orientation: landscape) and (max-height: 479px) {
+			margin-top: 12 * @px_base;
+		}
+		@media (orientation: portrait) and (min-width: 600px) {
+			margin-top: 24 * @px_base;
+		}
+		@media (orientation: landscape) and (min-width: 960px) {
+			margin-top: 12 * @px_base;
+		}
+	}
+
+	.ui-popup-controlbox {
+		height: auto;
+		margin-top: 8 * @px_base;
+		margin-left: -4 * @px_base;
+	}
+
+	.ui-popup-2level-description {
+		height: auto;
+		margin-top: 8 * @px_base;
+		margin-left: 8 * @px_base;
+		color: @color_popup_2level_description;
```

Comment:
Please use `var(...)` instead `less` variable

Reply:
We can use: var(--text-secondary-color)

Reply:
ok. done.